Mint.css

```diff
@@ -0,0 +1,196 @@
+body {
+  margin: 0;
+  padding: 0;
+  background: #eceef1;
+  box-sizing: border-box;
+}
+
+header {
+  box-sizing: border-box;
+}
+
+header p {
+  display: block;
+  float: right;
+  margin: auto;
+  padding-top: 10px;
+}
+
+header button {
+  color: white;
+  float: right;
+  margin-top: 10px;
+  margin-left: 5px;
+  margin-right: 20px;
+  border-radius: 5px;
+  border: 1px solid #4c4c4c;
+  width: 90px;
+  height: 40px;
+  background: #0077c5;
+}
+
+form button {
+  border: 1px solid #c1c5c8;
+  height: 44px;
+  width: 340px;
+  min-width: 160px;
+  margin-top: 30px;
+  padding-left: 9px;
+  padding-right: 6px;
+  background: #0077c5;
+  color: white;
+  outline: none;
+  font-size: 1.2em;
+  -webkit-border-radius: 3px;
+  border-radius: 4px;
+  line-height: 1.3em;
+}
+
+header button:hover {
+  background-color: green;
+}
+
+section {
+  display: block;
+  box-shadow: none;
+  text-align: center;
+  padding-top: 80px;
+}
+
+form {
+  margin: auto;
+  padding: 30px;
+  overflow: visible;
+  width: 350px;
+  display: block;
+  border: 1px solid #c7c7c7;
+  border-radius: 2px;
+  background: white;
+  height: auto;
+}
+
+form h2 {
+  color: #4c4c4c;
+  font-weight: 400;
+  line-height: 1.1;
+  font-size: 24px;
+  font-family: "AvenirNext", Avenir, "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
You are also repeating this property, please follow the DRY rule here too

Reply:
ok done